bugbug/model.py

```diff
@@ -65,12 +65,25 @@ def train(self, importance_cutoff=0.15):
         classes, class_names = self.get_labels()
         class_names = sorted(list(class_names), reverse=True)
 
+        # flag is true only for the duplicate model
+        flag = False
```

Comment:
I'm not sure you need this, if you make get_id of the BugCoupleModel return the tuple of bugs.